processes/fulfillURLManifest.py

```diff
@@ -62,31 +62,37 @@ def update_manifest(self, metadataObject, bucketName, currKey):
 
         metadataJSON = json.loads(metadataObject['Body'].read().decode("utf-8"))
         metadataJSONCopy = copy.deepcopy(metadataJSON)
+
+        #Checking if record asscociated with manifest is public domain or in copyright
+        checkRightsStatus = 'in copyright'
```

Comment:
the variable name is also not clear

Reply:
I changed the variable to rights_status to make it more clear.